drivers/openstack/openstack.go

```diff
@@ -555,7 +565,7 @@ func (d *Driver) resolveIds() error {
 		}).Debug("Found image id using its name")
 	}
 
-	if d.FloatingIpPool != "" {
+	if d.FloatingIpPool != "" && d.IpPoolType != "nova" {
```

Comment:
Scratch that - I wasn't paying attention to this. Carry on... Nothing to see here... :P